schemas/answers/text_field.json

```diff
@@ -26,6 +26,9 @@
       "mandatory": {
         "type": "boolean"
       },
+      "visible": {
```

Comment:
It would be good to add this to an existing valid schema under `tests/`

Reply:
This key should be part of detail answers and not text field?

Reply:
I've added one to `test_placeholder_source_ids.json`. It passes fine. Thanks!